src/generic/graphicc.cpp

```diff
@@ -527,7 +527,22 @@ class WXDLLIMPEXP_CORE wxCairoContext : public wxGraphicsContext
     int m_mswStateSavedDC;
 #endif
 #ifdef __WXGTK__
+    // This factor must be applied to the font before actually using it, for
+    // consistency with the text drawn by GTK itself.
     float m_fontScalingFactor;
+
+    // Function returning Pango font description for the given font scaled by
+    // the font scaling factor if necessary.
```

Comment:
Suggest: // the system defined font scaling factor if necessary (see GDK_DPI_SCALE).